packages/tc-ui/src/primitives/simple-relationship/server.jsx

```diff
@@ -0,0 +1,75 @@
+const React = require('react');
+const { getType } = require('@financial-times/tc-schema-sdk');
+const { WrappedEditComponent } = require('../../lib/components/input-wrapper');
+const { RelationshipPicker } = require('../relationship-picker');
+const { SelectedRelationship } = require('./lib/selected-relationship');
+
+const RelationshipPickerContainer = props => (
+	<div>
+		<RelationshipPicker
+			{...props}
+			SelectedRelationship={SelectedRelationship}
+			componentName="simple-relationship-picker"
+		/>
+	</div>
+);
+
+const {
+	ViewRelationship,
+	setRelationshipAnnotator,
+} = require('./lib/view-relationship');
+
+const maybeSort = (hasMany, props) => {
+	if (!hasMany) {
+		return '';
+	}
+	const sortField = props.has('name') ? 'name' : 'code';
+	return `(orderBy: ${sortField}_asc)`;
+};
+
+module.exports = {
+	name: 'SimpleRelationship',
+	ViewComponent: ViewRelationship,
+	EditComponent: props => (
+		<WrappedEditComponent
+			Component={RelationshipPickerContainer}
+			componentType="simple-relationship"
+			{...props}
+			value={props.value || null}
+		/>
+	),
+	parser: value => {
+		if (!value) {
+			return null;
+		}
+		value = JSON.parse(value);
+		// TODO use hasValue
+		if (!value) {
+			return null;
+		}
+		return Array.isArray(value)
+			? value.map(({ code }) => code)
+			: value.code;
```

Comment:
Should this also be an array so that the parser only returns one type? i.e. `[ value.code ]`

Reply:
no, because for hasMany fields the values are `[]` or `[some, values]`, whereas for non-hasMany they are a code (string) or null. 

In time the logic here (and in many other places) could do with branching once based on a clear criteria such as `if (hasMany) { ... }`, but not aiming to do that in this PR